### PR TITLE
docs(cn): importing and exporting components

### PR DESCRIPTION
**Pull request by Alienover**

### CODE REVIEW

File: src/pages/learn/importing-and-exporting-components.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+* 何时使用默认和命名导入导出
+* 如何在一个文件里导入和导出多个组件
+* 如何将组件拆分成多个文件
 
 </YouWillLearn>
 
 ## The root component file
 
-In [Your First Component](/learn/your-first-component), you made a `Profile` component and a `Gallery` component that renders it:
+在 [你的第一个组件](/learn/your-first-component) 中，你创建了一个 `Profile` 组件，并且渲染在 `Gallery` 组件里。
```
Comment (Alienover) [inline src/pages/learn/importing-and-exporting-components.md:23]: Will update the `Your First Component` after some translated it.

File: src/pages/learn/importing-and-exporting-components.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-* How to split components into multiple files
+* 何为根组件
+* 如何导入和导出一个组件
+* 何时使用默认和命名导入导出
+* 如何在一个文件里导入和导出多个组件
+* 如何将组件拆分成多个文件
 
 </YouWillLearn>
 
 ## The root component file
```
Comment (KnowsCount) [inline src/pages/learn/importing-and-exporting-components.md:21]:
```suggestion
## 根组件文件 {#the-root-component-file}
```

File: src/pages/learn/importing-and-exporting-components.md [hunk 3]
```diff
@@ -104,76 +104,76 @@ img { margin: 0 10px 10px 0; height: 90px; }
...
-<DeepDive title="Default vs Named Exports">
+<DeepDive title="默认导出 vs 命名导出">
 
-There are two primary ways to export values with JavaScript: default exports and named exports. So far, our examples have only used default exports. But you can use one or both of them in the same file. **A file can have no more than one _default_ export, but it can have as many _named_ exports as you like.**
+这是 JavaScript 里两个主要用来导出值的方法：默认导出和命名导出。到目前为止，我们的例子里仅仅是用到了默认导出。但你可以在一个文件里，使用其中一种或两种都使用。**一个文件里有且仅有一个 _默认_ 导出，但是可以有任意多个 _命名_ 导出。**
 
 ![Default and named exports](/images/docs/illustrations/i_import-export.svg)
 
-How you export your component dictates how you must import it. You will get an error if you try to import a default export the same way you would a named export! This chart can help you keep track:
+你组件的导出方式决定了你导入它的方式。当你用默认导入的方法去导入一个命名导出的组件，这时会有报错。下面的表格可以帮助你更好地理解：
```
Comment (KnowsCount) [inline src/pages/learn/importing-and-exporting-components.md:135]:
```suggestion
你组件的导出方式决定了你导入它的方式。当你用默认导入的方法去导入一个命名导出的组件，就会出现报错。你可以通过下面的表格来更好地理解：
```

File: src/pages/learn/importing-and-exporting-components.md [hunk 4]
```diff
@@ -104,76 +104,76 @@ img { margin: 0 10px 10px 0; height: 90px; }
...
+| 默认  | `export default function Button() {}` | `import Button from './button.js';`     |
+| 命名  | `export function Button() {}`         | `import { Button } from './button.js';` |
 
-When you write a _default_ import, you can put any name you want after `import`. For example, you could write `import Banana from './button.js'` instead and it would still provide you with the same default export. In contrast, with named imports, the name has to match on both sides. That's why they are called _named_ imports!
+当你用默认导入时，你可以在 `import` 语句后面起任何的名字。比如 `import Banana from './button.js'`，这样你还是能得到跟默认导出一样的内容。相反，对于命名导入，导入和导出的名字必须一致。这也是为什么叫 _命名_ 导入的原因！
 
-**People often use default exports if the file exports only one component, and use named exports if it exports multiple components and values.** Regardless of which coding style you prefer, always give meaningful names to your component functions and the files that contain them. Components without names, like `export default () => {}`, are discouraged because they make debugging harder.
+**通常文件里只有一个组件时，人们会使用默认导出，而当文件里有多个组件或值需要导出时，则使用命名导出。** 不管你使用哪种方式，请记得给你的组件和相应的文件起一个有意义的名字。我们不建议创建没有命名的组件，比如 `export default () => {}`，因为这样会使得调试的时候更加困难。
```
Comment (KnowsCount) [inline src/pages/learn/importing-and-exporting-components.md:144]:
```suggestion
**通常文件里只有一个组件时，人们会使用默认导出，而当文件里有多个组件或值需要导出时，则使用命名导出。** 不管你使用哪种方式，请记得给你的组件和相应的文件起一个有意义的名字。我们不建议创建没有命名的组件，比如 `export default () => {}`，因为这样会使得调试变得更加困难。
```

File: src/pages/learn/importing-and-exporting-components.md [hunk 5]
```diff
@@ -104,76 +104,76 @@ img { margin: 0 10px 10px 0; height: 90px; }
...
 
 </DeepDive>
 
-## Exporting and importing multiple components from the same file
+## 从一个文件里导出和导入多个组件 {#exporting-and-importing-multiple-components-from-the-same-file}
 
-What if you want to show just one `Profile` instead of a gallery? You can export the `Profile` component, too. But `Gallery.js` already has a *default* export, and you can't have _two_ default exports. You could create a new file with a default export, or you could add a *named* export for `Profile`. **A file can only have one default export, but it can have numerous named exports!**
+假如你想展示仅仅一个 `Profile` 而不是一整个图集？你也可以导出 `Profile` 组件。但 `Gallery.js` 里已经有一个 *默认* 导出了，你不能定义 _两个_ 默认导出。你可以在一个新文件里用默认导出，或者将 `Profile` 用 *命名* 导出。**一个文件里，有且仅有一个默认导出，但可以有多个命名导出！**
```
Comment (KnowsCount) [inline src/pages/learn/importing-and-exporting-components.md:150]:
```suggestion
假如你只想展示一个 `Profile` 而不展示整个图集呢？你也可以导出 `Profile` 组件。但 `Gallery.js` 里已经有一个 *默认* 导出了，你不能定义 _两个_ 默认导出。你可以在一个新文件里用默认导出，或者将 `Profile` 用 *命名* 导出。**一个文件里，有且仅有一个默认导出，但可以有多个命名导出！**
```

**Comment by Alienover** [comment]
@KnowsCount Thx for your review

**Comment by watonyweng** [comment]
@QC-L

**Comment by Alienover** [comment]
Close and move to the branch based on `main`

**Pull request closed without merging**